helm-charts/orchestrator/values.yaml

```diff
@@ -23,6 +23,7 @@ rhdhOperator:
   enabled: true # whether the operator should be deployed by the chart
   enableGuestProvider: false # whether to enable guest provider
   catalogBranch: v1.3.x # The branch for https://github.com/parodos-dev/workflow-software-templates used to import software templates resources
+  namespace: # Namespace for Existing RHDH Instance
```

Comment:
not sure if the variable name is clear enough. main issue is with the picked name for the top element `rhdhOperator` that refers to the operator but not to its instances, but that something will have to live with and we can address this in a nicer way with the go-based operator.
if we deal with an existing RHDH instance, should we specify its name as well? in other words, are we using the Backstage CR name anywhere other than creating the instance?

Reply:
When Jordi and I reviewed this addition, we saw how it could be confusing to the user and yes this is something we will address with the go operator. And we also have CRD spec proposal that Jordi is working on currently.
iirc, we are not using the backstage CR name anywhere else other than creating the instances. We mostly need the namespace so we can grab the configmap and update it. But Jordi might know better, I might be missing something.

Reply:
Yeah, I like the more verbose name 👍🏾  Thank you. Updated it [here](https://github.com/parodos-dev/orchestrator-helm-operator/pull/320/commits/ac0146416a96b0d9fcdf08ac86969cf1169c81a7) and also updated the `values-schema.json `file with the new field. I had missed that initially.